utils/build/rollup.config.js

```diff
@@ -1,6 +1,6 @@
 import babel from '@rollup/plugin-babel';
-import { terser } from 'rollup-plugin-terser';
-import babelrc from './.babelrc.json';
+import terser from '@rollup/plugin-terser';
+import babelrc from './.babelrc.json' assert { type: 'json' };
```

Comment:
In what versions of Node does that work?

Reply:
Thank you for your reply.

> In what versions of Node does that work?

```bash
node -v
v16.15.1

npm run dev
```

It will have an error without `assert { type: 'json' }`.

```
[ROLLUP] [!] TypeError: Module "file:///C:/github/three.js/utils/build/.babelrc.json" needs an import assertion of type "json"
[ROLLUP] TypeError: Module "file:///C:/github/three.js/utils/build/.babelrc.json" needs an import assertion of type "json"
[ROLLUP]     at new NodeError (node:internal/errors:372:5)
[ROLLUP]     at validateAssertions (node:internal/modules/esm/assert:82:15)
[ROLLUP]     at defaultLoad (node:internal/modules/esm/load:24:3)
[ROLLUP]     at ESMLoader.load (node:internal/modules/esm/loader:407:26)
[ROLLUP]     at ESMLoader.moduleProvider (node:internal/modules/esm/loader:326:22)
[ROLLUP]     at new ModuleJob (node:internal/modules/esm/module_job:66:26)
[ROLLUP]     at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:345:17)
[ROLLUP]     at ESMLoader.getModuleJob (node:internal/modules/esm/loader:304:34)
[ROLLUP]     at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:82:21)
[ROLLUP]     at async Promise.all (index 2)
[ROLLUP]
```

I also have try it in node `v18.12.0` LTS without `assert { type: 'json' }`, it also has this an error.

Reply:
> Is this because of the `assert` statement?

I got the same error with `node.js v16.13.2`, I tried to removed `assert { type: 'json' }` code, it does not success.

Reply:
Maybe it's a bit prematurely to use rollup 3.0 if we end up with such runtime errors. 

Besides, I wanted to suggest to remove babel from the build at the end of the year. With `r148` we want to remove `examples/js` which is probably a good opportunity to remove support for older browsers in general. That means we stop transpiling the core when building `three.js`, `three.cjs` and `three.min.js`.

Reply:
I fixed the error in node.js v16.13.2 with the following code:

```js
import fs from 'node:fs';
import path from 'node:path';
import url from 'node:url';

const __dirname = path.dirname( url.fileURLToPath( import.meta.url ) );

const toAbsolute = ( p ) => path.resolve( __dirname, p );

const babelrc = JSON.parse( fs.readFileSync( toAbsolute( '.babelrc.json' ), 'utf-8' ) );
```

Reply:
Thanks, but it's probably better to wait until next month when doing the `examples/js` changes. Depending on how the build is simplified, upgrading rollup will be easier. Closing for now.

Reply:
@Mugen87 It is time to reopen this PR?

Reply:
I'm not sure we need to upgrade Babel. If we plan to remove it anyway in a few month, I probably would not touch the dependency anymore.

BTW: We do not necessarily have to couple the removal of Babel with the removal of `three.min.js`, `three.js` and `three.cjs`. Since we've agreed to only use ES2018, we can probably remove it sooner?